src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorConstants.java

```diff
@@ -1,8 +1,27 @@
 package org.broadinstitute.hellbender.tools.funcotator;
 
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
 public class FuncotatorConstants {
     /**
      * Datasource name to use for Funcotations created from input variants from a VCF.
      */
-    public static String DATASOURCE_NAME_FOR_INPUT_VCFS = "INPUT_VCF";
+    public static final String DATASOURCE_NAME_FOR_INPUT_VCFS = "INPUT_VCF";
```

Comment:
Does this do dynamic differences in the protein changes?  In other words, if I have an input file that is mixed between mitochondria and non-mitochondria, will it adjust the protein changes appropriately?  If so, that is cool.

Reply:
Yes.

The mechanism for determining which Amino Acid Coding to use is pretty simplistic, but it works to differentiate the Mitochondria vs "normal" case.